src/Administration/Resources/app/administration/package.json

```diff
@@ -94,7 +94,7 @@
     "ts-jest": "29.0.3",
     "ts-loader": "8.4.0",
     "twig": "1.15.4",
-    "uuid": "8.3.2",
```

Comment:
Currently there is no such mecanism in place 🤔 IMHO this change is fine in a minor. Since we hide the package API behind our `util.service` createId. You shouldn't use our dependencies directly anyway.